kbase-extension/static/kbase/js/kbwidget.js

```diff
@@ -59,14 +59,14 @@ return KBWidget(
 
 Instantiate as follows:
 
-var $fancy = $('some-jquery-selector').myFancyNewWidget(
+const $fancy = $('some-jquery-selector').myFancyNewWidget(
```

Comment:
Did a big search for `var` and replaced them all with `let`, and then ran `eslint --fix`, which caught some of the `let`s that could be changed to `const`s. Fixed the rest by hand.